osbenchmark/worker_coordinator/runner.py

```diff
@@ -1848,32 +1849,43 @@ def __repr__(self, *args, **kwargs):
         return "delete-async-search"
 
 
-class OpenPointInTime(Runner):
+class CreatePointInTime(Runner):
     async def __call__(self, opensearch, params):
         op_name = mandatory(params, "name", self)
         index = mandatory(params, "index", self)
         keep_alive = params.get("keep-alive", "1m")
-        response = await opensearch.open_point_in_time(index=index,
-                                         params=params.get("request-params"),
-                                         keep_alive=keep_alive)
+        response = await opensearch.create_point_in_time(index=index,
+                                                         params=params.get("request-params"),
+                                                         keep_alive=keep_alive)
         id = response.get("id")
         CompositeContext.put(op_name, id)
 
     def __repr__(self, *args, **kwargs):
-        return "open-point-in-time"
+        return "create-point-in-time"
 
 
-class ClosePointInTime(Runner):
+class DeletePointInTime(Runner):
```

Comment:
I remember us discussing to have deleteAllPointInTime outside composite context - lets see if we can add that as well to code repository.